packages/util-cli-reporter/lib/js/reporter.js

```diff
@@ -81,26 +87,30 @@ report.info = (description, message = null, comment = null) => {
  * @param {String} [comment=null] additional comment
  */
 report.success = (description, message = null, comment = null) => {
-	console.log(
-		cleanWhitespace(
-			configureOutput('success', description, message, comment)
-		)
-	);
+	if (logLevel >= 2) {
```

Comment:
Can you use constants for the log levels?
`const SUCCESS = 2;` etc

Reply:
how do you see that working?

Reply:
declare the `const`'s at the top
```
const INFO = 1;
const SUCCESS = 2;
const ERROR = 3;
```

Then use them instead of the numbers in the code.
```
if (logLevel >= SUCCESS) {
```

Reply:
yep, and setting `logLevel`? if I run `reporter.init('success')` I would have to check if that variable exists (typeof I guess?) and set that based on that. Bit more verbose.

Reply:
It would just be for readability in this file.